xdem/coreg.py

```diff
@@ -585,27 +586,40 @@ def apply(self, dem: np.ndarray | np.ma.masked_array | RasterType,
         # See if a _apply_func exists
         try:
             # Run the associated apply function
-            applied_dem = self._apply_func(dem_array, transform)  # pylint: disable=assignment-from-no-return
+            applied_dem = self._apply_func(dem_array, transform, **kwargs)  # pylint: disable=assignment-from-no-return
         # If it doesn't exist, use apply_matrix()
         except NotImplementedError:
             if self.is_affine:  # This only works on it's affine, however.
+                # If dilate_mask is not specified, set it to True by default
```

Comment:
Nice to have this option accessible through kwargs!

Reply:
This was actually the initial intention of this PR !